packages/foreman/foreman/foreman.spec

```diff
@@ -759,6 +762,33 @@ install -Dp -m0644 %{SOURCE5} %{buildroot}%{_tmpfilesdir}/%{name}.conf
 install -Dp -m0644 extras/systemd/%{name}.service %{buildroot}%{_unitdir}/%{name}.service
 install -Dp -m0644 extras/systemd/%{name}.socket %{buildroot}%{_unitdir}/%{name}.socket
 
+# SELinux libexec wrappers
+%if 0%{?scl:1}
+  cat > %{buildroot}%{_libexecdir}/%{name}/ruby-selinux <<-EOF
+  #!/bin/bash
+  # Shell wrapper with SELinux transition into foreman_rails_t domain.
+  source scl_source enable %{scl_name}
+  exec ruby "\$@"
+EOF
+  cat > %{buildroot}%{_libexecdir}/%{name}/sidekiq-selinux <<-EOF
+  #!/bin/bash
+  # Shell wrapper with SELinux transition into foreman_rails_t domain.
+  source scl_source enable %{scl_name}
+  exec sidekiq "\$@"
```

Comment:
The built packages contain unexpanded `%{scl_name}`. Is it the right macro?

```
[root@foreman-selinux foreman]# cat /usr/libexec/foreman/ruby-selinux
  #!/bin/bash
  # Shell wrapper with SELinux transition into foreman_rails_t domain.
  source scl_source enable %{scl_name}
  exec ruby "$@"
[root@foreman-selinux foreman]# /usr/libexec/foreman/ruby-selinux -e 'puts "Hello"'
Can't read /etc/scl/conf/%{scl_name}, %{scl_name} is probably not installed.
Hello
```

Reply:
It's just `%{scl}`

Reply:
That's indeed incorrect, let me check.

Reply:
Replaced with scl_prefix that should do it.